sources/Monaspace Xenon Var.glyphs

```diff
@@ -53,6 +53,10 @@ value = 0;
 {
 name = "Use Typo Metrics";
 value = 1;
+},
+{
+name = isFixedPitch;
```

Comment:
That would be automatically set if it would be valid iirc.

Forcing the value to `1` while the font is not `isFixedPitch` can lead to problems, see for example https://github.com/google/fonts/issues/225

Anyhow, the root cause should be in
* #132 

and supposedly it is fixed now. But it is not. I can redo the search for offending glyphs if need be.

Here the isFixedPitch value in the font file:

```
fini@Air ~ % showttf ~/Downloads/monaspace-v1.101/fonts/otf/MonaspaceArgon-Regular.otf | tail -n +1065 | head
Required tables: glyf and loca have been replaced by CFF => OpenType
post table (at 7312)
	 format=00030000
	 italicAngle=0
	 underlinePos=-70
	 underlineWidth=72
	 fixedpitch=0
	 mem1=0
	 mem2=0
	 mem3=0
	 mem4=0
fini@Air ~ %
```

Here a screenshot of the "warning" to set it in Glyphs ;-)

<img width="1470" alt="Screenshot 2024-06-07 at 17 13 21" src="https://github.com/githubnext/monaspace/assets/16012374/047e4aa0-2632-48e4-bdbd-7b0a01ebf17d">